llvm/test/MC/Disassembler/X86/apx/ldtilecfg.txt

```diff
@@ -0,0 +1,6 @@
+# RUN: llvm-mc --disassemble %s -triple=x86_64 | FileCheck %s --check-prefixes=ATT
```

Comment:
I still think file/instruction brings too much files, can we organize in CPUID of being promoted instructions, e.g., amx-tile.txt?

Reply:
Agree, I think the impl would be cleaner, will do. @KanRobert, these promoted instrs usually only have 1 form for each instrs, it may be doable, and for those has nf or ndd version, we add them one test for each instr, what do you think?

Reply:
Good suggestion. I agree with you and phoebe, we can classify the instruction by CPUID if there is no NDD and NF variant, For NDD, NF instructions, we can keep using the mnemonic.